README.md

```diff
@@ -1,19 +1,77 @@
 <br />
 <p align="center">
-  This repository contains a fully usable <a href="https://github.com/BookStackApp/BookStack"><b>BookStack</b></a> with <a href="https://github.com/traefik/traefik"><b>Traefik</b></a> as a reverse proxy. Additionally, Traefik uses <a href="https://letsencrypt.org/"><b>Let's Encrypt</b></a> as an ACME provider to automate certificate generation. The whole stack can be set up with the included <a href="https://github.com/ansible/ansible"><b>Ansible</b></a> code.
+  This repository contains a fully usable <a href="https://github.com/BookStackApp/BookStack"><b>BookStack</b></a> with <a href="https://github.com/traefik/traefik"><b>Traefik</b></a> as a reverse proxy. Additionally, Traefik uses <a href="https://letsencrypt.org/"><b>Let's Encrypt</b></a> as an ACME provider to automate certificate generation.
 </p>
 <br />
 
 ## ⚡️ Installation
 
 1. Make sure you've installed Docker including `docker-compose` support on your target system.
+   You can install `docker-compose` following this instruction: https://docs.docker.com/compose/install/
+
+<br>
 
 2. Clone this repository:
 
    ```bash
-   git clone https://github.com/symnixhq/traefik-bookstack.git
+   sudo git clone https://github.com/symnixhq/traefik-bookstack.git
+   ```
+
+<br>
+
+3. Open the file called `traefik-letsencrypt-bookstack-docker-compose.yml`
+
+<br>
+
+4. Adjust the variables for your configuration. Edit the following values:
+   1. Passwords (declared as "MODIFY_ME")
+   2. E-Mail address (declared as "example.mail@mail.com")
```

Comment:
Is mail.com a reserved example domain, like example.com, or could this end up in a real inbox?

Reply:
I've updated the example mail to something non-existent. Thanks for the hint.